src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
+    result: 0,
+
+    operate(callback, x) {
+      this.result = callback(this.result, x);
+
+      return this;
+    },
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    divide(a, b) {
+      return a / b;
+    },
 
+    multiply(a, b) {
+      return a * b;
+    },
+
+    reset() {
+      calculator.result = 0;
+
+      return calculator;
```

Comment:
you should use `this.result` instead of `calculator.result` and return `this `instead of `calculator`, when you code methods